src/containers/RootPage.js

```diff
@@ -122,15 +122,14 @@ class RootPage extends PureComponent {
       }
     }
 
-
-    if (this.props.match.url.indexOf(`/hub`) === 0) {
+    if (this.props.match.url === (`/hub`)) {
       doRootPage.markLogSeen(myId)
     }
     else {
       doRootPage.getLogLastSeen(myId)
     }
 
-    if (this.props.match.url.indexOf(`/friend`) === 0) {
+    if (this.props.match.url === `/friend`) { // only index send
       doRootPage.markFriendListSeen(myId)
```

Comment:
Do we still need this if markFriendListSeen is called in componentDidMount in container/FriendListPage.js ?

Reply:
Yes, I add `markFriendListSeen` in `container/FriendListPage.js` just because of speed.

The `markFriendListSeen` here is used for refresh, which means page should be marked seen as long as user key staying in this page.

Reply:
got it thanks.